clang/lib/Driver/ToolChains/CommonArgs.cpp

```diff
@@ -2402,7 +2402,7 @@ void tools::checkAMDGPUCodeObjectVersion(const Driver &D,
 
 unsigned tools::getAMDGPUCodeObjectVersion(const Driver &D,
                                            const llvm::opt::ArgList &Args) {
-  unsigned CodeObjVer = 4; // default
+  unsigned CodeObjVer = 5; // default
```

Comment:
Perhaps at least this could be avoided? The flag configuration already says what th edefault value is - could that be relied on without needing to specify a value here too?